app/registration/views/patient-summary.html

```diff
@@ -3,7 +3,7 @@
         <li role="presentation" data-toggle="tab" class="active"><a ui-sref="dashboard.program">{{'COMMON_PROGRAM_ENROLLMENT_TITLE' | translate}}</a></li>
         <li role="presentation" data-toggle="tab" class="active"><a ui-sref="dashboard.visits">{{'COMMON_VISIT_HISTORY_TITLE' | translate}}</a></li>
         <li role="presentation" data-toggle="tab" class="active"><a ui-sref="dashboard.services">{{'COMMON_CLINIC_SERVICES_TITLE' | translate}}</a></li>
-        <li role="presentation" data-toggle="tab" class="active"><a ui-sref="dashboard.alerts">{{'COMMON_ALERT_TITLE' | translate}}</a></li>
+        <!--<li role="presentation" data-toggle="tab" class="active"><a ui-sref="dashboard.alerts">{{'COMMON_ALERT_TITLE' | translate}}</a></li>-->
```

Comment:
We should decide whether or not alerts will be presented in registration. Stuff like for e.g. : this patient is early/late for consultation, this person is deceased, This patient has his next consultation in 3 days, etc.